cloud/monitoring/groups.md

```diff
@@ -31,11 +31,14 @@ We also try to group services by application jobs. Example: Salary management ap
 
 ## Creating a host group
 
-Go to the **Configuration > Hosts > Host Groups (simplified)** menu and click **Add**
+Go to the **Configuration > Hosts > Host Groups** menu and click **Add**
 
 * The **Name** and **Alias** defines the name and the alias of the host group.
-* The **Geographic coordinates** field defines geographical coordinates used by the Centreon MAP module to position the resource on a map
-* The **Members** list allows us to add hosts in the hostgroup.
+* The **Group members** field allows us to add hosts in the hostgroup.
+* The [**Resource access rules**](../administration/resource_access.md) allows us to grant the group access to the resources.
+* The **Geographic coordinates** field defines geographical coordinates used by the Centreon MAP module to position the resource on a map.
+* The selected **Icon** will be displayed in the list of host groups.
+* The **Comments** allows us to make comments about the group.
```

Comment:
- Do not use "allows us to". "us" sounds odd. if you really need to use "allows", write "allows users to"
but in this case it is not necessary.
- There is no need to state that the comments fields is used to enter comments. The only useful thing you can say is probably "Comments entered here are for internal reference only and will only be visible on this page." or, more synthetically, "Notes added here are only visible on this page".